.kokoro/build.sh

```diff
@@ -119,5 +128,8 @@ if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"release"* ]]; then
 fi
 
 npm test
```

Comment:
This seems like the wrong approach. Let's use `npm run --if-present system-test` instead of making this project specific.